libpod/runtime_img.go

```diff
@@ -136,6 +142,17 @@ func (r *Runtime) RemoveImage(ctx context.Context, image *image.Image, force boo
 	return image.ID(), err
 }
 
+// IsParent goes through the layers in the store and checks if "layer" is
+// the parent of any other layer in store.
+func (r *Runtime) IsParent(storageLayers []storage.Layer, layer string) bool {
```

Comment:
Can we make this a function on the image in question?

Reply:
@mheon are you thinking this goes into the image library?  It might also be good to have a GetParent ... etc

Reply:
@baude Yeah I was thinking this was image library stuff - IsParent(), GetParent() all strike me as good image methods